tests/casr_tests/test_casr_ignore_frames/ign1.lst

```diff
@@ -0,0 +1,2 @@
+FUNCTIONS
```

Comment:
We need documentation for ign.lst format. Update usage.md.

Reply:
done